concent_api/core/models.py

```diff
@@ -684,3 +685,45 @@ class PendingEthereumTransaction(Model):
     s = DecimalField(max_digits=78, decimal_places=0)
 
     created_at = DateTimeField(auto_now_add=True)
+
+
+class DepositAccount(Model):
+    client = ForeignKey(Client)
+    ethereum_public_key = CharField(max_length = ETHEREUM_ADDRESS_LENGTH)
+    created_at = IntegerField()
```

Comment:
`created_at` and `modified_at` should be:
```
    created_at = DateTimeField(auto_now_add=True)
    modified_at = DateTimeField(auto_now=True)
```

Reply:
Done.

Reply:
@rwrzesien, please update the issue: timestamp is a bit misleading - datetime seems a bit more clear ;)